Src/AutoFixtureUnitTest/DataAnnotations/StringLengthArgumentSupportTests.cs

```diff
@@ -24,15 +24,33 @@ public void FixtureCorrectlyCreatesShortText()
             // Teardown
         }
 
+        [Fact]
+        public void FixtureCorrectlyCreatesLongText()
+        {
+            // Fixture setup
+            var fixture = new Fixture();
+            // Exercise system
+            var actual = fixture.Create<ClassWithLengthConstrainedConstructorArgument>();
+            // Verify outcome
+            Assert.Equal(
+                ClassWithLengthConstrainedConstructorArgument.LongTextLength,
+                actual.LongText.Length);
+            // Teardown
+        }
+
         private class ClassWithLengthConstrainedConstructorArgument
         {
             public const int ShortTextMaximumLength = 3;
+            public const int LongTextLength = 50;
             public readonly string ShortText;
+            public readonly string LongText;
 
             public ClassWithLengthConstrainedConstructorArgument(
-                [StringLength(ShortTextMaximumLength)]string shortText)
+                [StringLength(ShortTextMaximumLength)]string shortText,
```

Comment:
How do the changes in this class impact existing tests? Could it cause existing tests to become false negatives?